WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1803,3 +1805,12 @@ FileTypeCal         = @FileIconCommonStyle@ background-color:#94385d;
 FileTypeVCard       = @FileIconCommonStyle@ background-color:#666666;
 FileTypeMedia       = @FileIconCommonStyle@ background-color:#9e61ca;
 FileTypeMail        = @FileIconCommonStyle@ background-color:@PrimaryBrandingColor@
+
+#######################################################################
+#   Create Contact-Group Dialog
+#######################################################################
+
+CreateContactGroupDialog        =
```

Comment:
`DialogBodyText` is very general properties that targets all content in dialog. If you see in zm.css. 

Reason I have added empty `CreateContactGroupDialog` is because I have added one selector `#CreateContactGroupDialog ` in zm.css. And for this we don't need any style as such for now.

But having property `CreateContactGroupDialog ` will enables other skin developer to override styles for CreateContactGroupDialog. Because if you see, sometimes we may need dialog specific styles that are different that common styles.

Let me know if you have any questions.

Thanks